repos/c2sm/packages/py-gridtools-cpp/package.py

```diff
@@ -29,7 +33,7 @@ class PyGridtoolsCpp(PythonPackage):
             sha256=
             "0bc4b72c9f2786f70c80092bba7bd2eedd216d8c16946b7b32704d248e0d97e6")
 
-    depends_on("python@3.8:")
+    depends_on("python@3.10:")
```

Comment:
this change is because `gt4py.next` has (always had) a minimum requirement of 3.10

Reply:
actually this is wrong...

Reply:
so what now?

Reply:
From my point of view is does not depend, gt4py anyway is built using python 3.10 with Spack